README.md

```diff
@@ -51,3 +51,13 @@ work with you.
 * [AngularDart documentation](https://angulardart.org/)
 * [Material Design site](https://material.google.com/)
 * [Code Lab: AngularDart Components](https://webdev.dartlang.org/codelabs/angular2_components)
+
+## Custom Component Styles
+
+Get access to the core Material Design SASS files used to style these components by copying them 
```

Comment:
Thanks. My intellij had a customized setting for line length of 100 columns after installing and I have just left it at that. I hand't realized that it was too long. I will update.